sql/alps_submitter.go

```diff
@@ -0,0 +1,167 @@
+package sql
+
+import (
+	"fmt"
+	"io"
+	"strconv"
+)
+
+type GraphConfig struct {
+	TableName string
+	Fields    []string
+	X         []FeatureSpec
+	Y         FeatureSpec
+}
+
+type FeatureSpec struct {
+	FeatureName string
+	IsSparse    bool
+	Shape       []int
+	DType       string
```

Comment:
Do we need a field `Separator` for `DENSE`?

Reply:
Fixed